root/static/pages/ddh_up_index.html

```diff
@@ -1,2 +1,296 @@
+    </ul>
+</div>
+
+
+    </div>
+</div>
+
+
+<div class="site-header mobile">
+  <div class="content-wrap">
+    <div class="flex-container">
+
+      <div class="header-logo">
+	<div class="header-logo__logo"><a href="/"><img src="/ddh-static/images/logo.svg" /></a></div>
+      </div>
+
+      <div class="nav-menu">
+	<a href="#">
+	  <i class="ddgsi ddgsi-menu"></i>
+	</a>
+      </div>
+    </div>
+  </div>
+</div>
+
+<div class="slideout">
+  <ul class="nav-menu__list">
+    <i class="ddgsi ddgsi-close"></i>
+    <li class="nav-menu__heading"><span>Menu</span></li>
+    <li class="nav-menu__item"><a href="/ia/dev">Instant Answers</a></li>
+    <li class="nav-menu__item"><a href="http://docs.duckduckhack.com/">Docs</a></li>
+    <li class="nav-menu__item"><a href="/help">Help</a></li>
+    <li class="nav-menu__item"><a href="/blog" class="active">Blog</a></li>
+
+      <li class="nav-menu__heading"><span>Account</span></li>
+      <li class="nav-menu__item"><a href="/my/register">Sign Up</a></li>
+      <li class="nav-menu__item"><a href="/my/login">Log In</a></li>
+  </ul>
+</div>
+        <div class="site-main">
+            <div class="">
+
+<script src="/ddh-static/js/userpages/angular.min.js"></script>
+<script src="/ddh-static/js/userpages/angular-sanitize.min.js"></script>
+<script src="/ddh-static/js/userpages/moment.min.js"></script>
+<script src="/ddh-static/js/userpages/underscore-min.js"></script>
+<script src="/ddh-static/js/userpages/app.js"></script>
+<script src="/ddh-static/js/userpages/UserPage.js"></script>
+
+<link rel="stylesheet" href="/ddh-static/css/userpages/main.css">
+<link rel="stylesheet" href="/ddh-static/fonts/userpages/octicons/octicons.css" charset="utf-8">
+<link rel="stylesheet" href="/ddh-static/css/userpages/UserPage.css">
+
+<title>{{response.gh_data.login}} | DuckDuckHack</title>
+
+<!-- Header -->
+<header>
+  <div class="container">
+    <div class="stats">
+      <div class="third">
+	<span>All Instant Answers</span>
+	<h2>{{count.total_ias}}</h2>
+	</div>
+      <!--
+      <div class="fourth">
+	<span>Assigned Pull Requests</span>
+	<h2>{{count.reviewed_prs}}</h2>
+	</div>
+      -->
+      <div class="third">
+	<span>Open Pull Requests</span>
+	<h2>{{count.developed_prs}}</h2>
+	</div>
+      <div class="fourth">
+	<span>Closed Pull Requests</span>
+	<h2>{{count.closed_prs}}</h2>
+	</div>
+      </div>
+</div></header>
+
+<div class="container">
+
+  <!-- Sidebar -->
+  <aside>
+    <div class="profile card">
+      <img src="{{response.gh_data.avatar_url}}" />
+      <div class="content">
+	<h3>{{response.gh_data.name}}</h3>
+	<div class="gray">
+	  <a href="https://github.com/{{response.gh_data.login}}">@{{response.gh_data.login}}</a>
+	  <br />{{response.gh_data.location}}
+	  </div>
+
+	</div>
+</div>
+    <div class="topic typeC" ng-repeat="topic in topics | orderBy:'-count'">
+      <span class="amt"> {{topic.count}}</span>
+      <span> {{topic.topic}}</span>
```

Comment:
Mr. Chris has a million topics, but they grow dangerously close to the footer of his page.Add some padding?

![screen shot 2016-04-13 at 7 54 35 am](https://cloud.githubusercontent.com/assets/4480279/14492402/fe471af6-014c-11e6-8321-545793d1acba.png)